designsafe/static/styles/app-card.css

```diff
@@ -0,0 +1,101 @@
+/* STRUCTURE */
+
+/* To load most structure from Core-Styles */
+@import url("https://cdn.jsdelivr.net/gh/TACC/core-styles@v2.25.0/dist/trumps/s-app-card.css");
+@import url("https://cdn.jsdelivr.net/gh/TACC/core-styles@v2.25.0/dist/settings/font.css");
+
+/* To support equal-height columns in Bootstrap 3 */
+@import url("https://cdn.jsdelivr.net/gh/TACC/core-styles@v2.25.0/dist/components/bootstrap.row.css");
+@import url("https://cdn.jsdelivr.net/gh/TACC/core-styles@v2.25.0/dist/settings/space.css");
```

Comment:
**Concern.** See https://github.com/DesignSafe-CI/portal/pull/1168#discussion_r1499941354.

Reply:
In [private (internal) communication](https://tacc-team.slack.com/archives/DRG6JLLCV/p1709316715623679), I've expressed safety of Core-Styles via CDN.

**Gist**: Core-Styles is used piecemeal, and adds **unique** global vars, **unique** component class name, a **unique** classname for a custom Bootstrap3 utility. Its global introduction would not conflict with DesignSafe.

Reply:
**Concern Clarified.**

1. loading multiple files
2. loading **all** (not true) of Core-Styles
3. worry that goal is to make DS match Core

**Solution Found.**

1. only 4, but I will reduce it to 2 (by copy-pasting settings I use, instead of importing many)
2. not so; only loading what is used
3. not so; the imported styles are [structure not skin](https://www.google.com/search?q=css+oocss+structure+skin)

Reply:
1. reduced imports to just 2 (`s-app-card`, `bootstrap.row`)

Resolving conversation. If there are more questions, just ask and un-resolve 🙂.